### DISCUSSION TITLE
Latest server version needs a new (unreleased) app version

### DISCUSSION THREAD

**Discussion by RayBomb87** [immich-app/immich#14508] [Q&A] [answered]
### The bug

I just installed the latest server version. 1.122.0
Now my mobile app says it out of date, but there is no new updates available. 

### The OS that Immich Server is running on

Unraid 7 - docker

### Version of Immich Server

V1.122.0

### Version of Immich Mobile App

V1.121.0 build 184

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Not needed for issue
```

### Your .env content

```Shell
Not needed for issue
```

### Reproduction steps

1.
2.
3.
...

### Additional information

![image](https://img.raylab.no/i/a430215d-8f2b-4663-8651-eff403939dbc.jpg)

![image](https://img.raylab.no/i/89cfa18b-2d9a-420d-88c3-315083bf029a.jpg)

**Comment by mertalev** [comment, marked as answer]
The new app version is still being reviewed. There are no breaking changes that actually require the app to be 1.122.0, so you can just update when it comes out.
